### PR TITLE
Upgrade Guide to 6.0

### PR DESCRIPTION
**Pull request by dbpolito**

### CODE REVIEW

File: source/assets/build/mix-manifest.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
     "/js/main.js": "/js/main.js?id=ce06880262eb35d7b51f",
-    "/css/main.css": "/css/main.css?id=54cadb4790223bda9a0b",
+    "/css/main.css": "/css/main.css?id=54b89e2eb922f2893d01",
```
Comment (nunomaduro) [inline source/assets/build/mix-manifest.json]: This pull request should only contain modifications on the file `upgrade.md`.

File: source/docs/upgrade.md [hunk 2]
```diff
@@ -7,10 +7,28 @@ section: content
...
+
+### Updating Dependencies
+
+Update your `laravel-zero/framework` dependency to `^6.0` in your `composer.json` file.
+
+#### If you are using Laravel's String & Array Helper
+
+All `str_` and `array_` helpers have been moved to the new `laravel/helpers` Composer package and removed from the framework. If desired, you may update all calls to these helpers to use the `Illuminate\Support\Str` and `Illuminate\Support\Arr` classes. Alternatively, you can add the new `laravel/helpers` package to your application to continue using these helpers:
+
+    composer require laravel/helpers
```
Comment (nunomaduro) [inline source/docs/upgrade.md]: This should be under markdown code.

**Review by owenvoke — APPROVED** [review]
LGTM!

**Review by nunomaduro — CHANGES_REQUESTED** [review]

File: source/docs/upgrade.md (hunk 2 above)
Comment (nunomaduro) [reply source/docs/upgrade.md]: Would be cool also a note to check the laravel upgrade guide,

**Comment by dbpolito** [comment]
@nunomaduro updated

**Pull request merged**